### ISSUE TITLE
Support for new Decks / custom Decks

### ISSUE THREAD

**Issue by gabeochoa** [gabeochoa/Sokim#2]
We should add support

**Comment by alanplotko** [comment]
Definitely. We just need an overall settings page prior to creating a room. A formal room class may be useful as well for more sophisticated and cleaner use across the game.

**Comment by gabeochoa** [comment]
How are we going to allow them to add the cards? maybe they can give an imgur album?

**Comment by alanplotko** [comment]
Pass in a valid album link (we can parse it to see if it's valid). We can then send it over to our image-fetching program for processing. We just have to be sure it won't crash the entire app.
